packages/git-tools/scripts/git-valu-push-production

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+if [ "$(git rev-parse --abbrev-ref HEAD)" != "master" ]; then
+  echo "You can deploy to production only from the local master branch"
```

Comment:
nit: Usually error messages (this and others) should be logged to stderr so piping would not hide them. We probably won't be piping these probably does not matter.

```suggestion
  >&2 echo "You can deploy to production only from the local master branch"
```